org.eclipse.scanning.points/src/org/eclipse/scanning/points/GridIterator.java

```diff
@@ -116,17 +108,11 @@ public GridIterator(RandomOffsetGridGenerator gen) {
         
         JythonObjectFactory compoundGeneratorFactory = ScanPointGeneratorFactory.JCompoundGeneratorFactory();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "compoundGeneratorFactory" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1481)